python/ray/rllib/envs.py

```diff
@@ -5,8 +5,11 @@
 import gym
```

Comment:
We actually just removed this file; It makes sense to rename this as vectorized_env or something along those lines.